README.md

```diff
@@ -3,31 +3,93 @@ The ID.me WebVerify SDK for iOS is a library that allows you to verify a user's
 
 ## Release Information
 
-- **SDK Version:** 3.2.0 (June 24, 2016)
+- **SDK Version:** 4.0.0 (March 28, 2017)
 - **Maintained By:** [ID.me](http://github.com/IDme)
 
 For more information please email us at mobile@id.me or visit us at http://developer.id.me.
 
-## Changelog (3.2.0)
-- Changed parameter from affiliationType to scope (NSString)
+## Changelog
+The changelog can be found in [CHANGELOG.md](https://github.com/IDme/ID.me-WebVerify-SDK-iOS/blob/master/CHANGELOG.md)
 
-## Download
+## Installation
 
 Get it using CocoaPods
 
 ```
 pod 'IDmeWebVerify'
 ```
 
-or download it on Github.
+or download it from Github and add it to your Xcode project.
 
 ## Setup
-1. Drag the **ID.me WebVerify SDK** folder into your project. This will add the following files:
-	- `IDmeWebVerify.h`
-	- `IDmeWebVerifyNavigationController.h`
-2. Import `IDmeWebVerify.h` into one of your projects, preferably an instance or subclassed instance of `UIViewController`.
+### Step 1
+If you installed the SDK through Cocoapods you can skip this step. Drag the downloaded files to your Xcode project. If you are working with Swift you will have to import `IDmeWebVerify.h` in your ObjC Bridging Header. You will also have to somehow import [SAMKeychain](https://github.com/soffes/SAMKeychain) to your project as this SDK depends on it.
+
+### Step 2
+You must call `IDmeWebVerify.initialize(withClientID: String, clientSecret: String, redirectURI: String)` before using the SDK. You can do it in `application(didFinishLaunchingWithOptions:)` for example.
```

Comment:
Maybe you can add an explanation on what are those parameters, https://github.com/IDme/ID.me-WebVerify-SDK-Android/pull/33/files#diff-04c6e90faac2675aa89e2176d2eec7d8R84